src/Helldivers-2-Models/ArrowHead/Status/PlanetEvent.cs

```diff
@@ -9,7 +9,8 @@ namespace Helldivers.Models.ArrowHead.Status;
 /// <param name="PlanetIndex">The index of the planet.</param>
 /// <param name="EventType">A numerical identifier that indicates what type of event this is.</param>
 /// <param name="Race">The identifier of the faction that owns the planet currently.</param>
-/// <param name="Health">The current health of the planet.</param>
+/// <param name="Health">The current health of the event.</param>
```

Comment:
The event is about a planet, so wasn't the original docs correct?

Reply:
Seeming as the health differs from the planet health i think the health was specific to the event?

Reply:
makes sense